src/XMLexport.cpp

```diff
@@ -423,6 +423,7 @@ void XMLexport::writeHost(Host* pHost, pugi::xml_node mudletPackage)
     host.append_attribute("mMapViewOnly") = pHost->mMapViewOnly ? "yes" : "no";
     host.append_attribute("mShowRoomIDs") = pHost->mShowRoomID ? "yes" : "no";
     host.append_attribute("mShowPanel") = pHost->mShowPanel ? "yes" : "no";
+    host.append_attribute("mShowVideoPlayerPanel") = pHost->mShowVideoPlayerPanel ? "yes" : "no";
```

Comment:
*Personally* I do not like to see internal variable names inside the XML game save files - XML is supposed to be readable by humans as well as machines, so I'd tend to call this something like:
```suggestion
    host.append_attribute("ShowVideoPlayerControlPanel") = pHost->mShowVideoPlayerPanel ? "yes" : "no";
```

Reply:
Let's stick with the approach most variables have used for now